configs/Ec2UserdataUtils.psm1

```diff
@@ -1293,30 +1277,33 @@ function Install-BundleClone {
   }
 }
 
-function Enable-BundleClone {
+function Enable-CloneBundle {
   param (
     [string] $hgrc = [IO.Path]::Combine([IO.Path]::Combine([IO.Path]::Combine(('{0}\' -f $env:SystemDrive), 'mozilla-build'), 'hg'), 'Mercurial.ini'),
-    [string] $path = [IO.Path]::Combine([IO.Path]::Combine([IO.Path]::Combine(('{0}\' -f $env:SystemDrive), 'mozilla-build'), 'hg'), 'bundleclone.py'),
     [string] $domain = $env:USERDOMAIN
   )
   begin {
     Write-Log -message ("{0} :: Function started" -f $($MyInvocation.MyCommand.Name)) -severity 'DEBUG'
   }
   process {
-    if ($domain.Contains('.usw2.')) {
-      $ec2region = 'us-west-2'
-    } else {
-      $ec2region = 'us-east-1'
+    switch -wildcard ($domain) 
+    { 
+        '*.usw2.*' {
+          $ec2region = 'us-west-2'
+        }
+        default {
+          $ec2region = 'us-east-1'
+        }
     }
     if (!(Test-Path (Split-Path $hgrc) -PathType Container)) {
       Write-Log -message ("{0} :: detected missing mercurial installation" -f $($MyInvocation.MyCommand.Name)) -severity 'ERROR'
     } else {
       Remove-Item -path $hgrc -force
       Out-IniFile -FilePath $hgrc -encoding 'UTF8' -InputObject @{
         "ui"=@{
-          "editor"='"C:\Program Files\Sublime Text 3\sublime_text.exe" --wait --new-window';
           "traceback"="True";
           "username"="Mozilla Release Engineering <release@mozilla.com>"
+          "clonebundleprefers"=('ec2region={0}, VERSION=packed1' -f $ec2region)
```

Comment:
The ec2region bit shouldn't be necessary: hg.mozilla.org looks at the source IP and advertises URLs for the local AWS region (if available) and prefers the CDN otherwise.